torch/_functorch/_aot_autograd/subclass_utils.py

```diff
@@ -92,7 +93,11 @@ def create_subclass_meta(
             start_idx = idx
             subclass_meta, _ = create_subclass_metadata(a, start_idx)
             infos.append(subclass_meta)
-            cnt = subclass_meta.arg_count
+            cnt = (
+                subclass_meta.arg_count
+                + count_symints * len(filter_symints(a.size()))
```

Comment:
I had to pause for a sec to understand the `count_symints * len(...)` here but I guess it saves us a few lines 😛

Reply:
I'll add a comment